x-pack/plugins/security_solution/public/overview/pages/detection_response.test.tsx

```diff
@@ -11,9 +11,26 @@ import { render } from '@testing-library/react';
 import { DetectionResponse } from './detection_response';
 import { TestProviders } from '../../common/mock';
 
+jest.mock('../components/detection_response/alerts_by_status', () => ({
+  AlertsByStatus: () => <div data-test-subj="mock_AlertsByStatus" />,
+}));
+
+jest.mock('../components/detection_response/cases_table', () => ({
+  CasesTable: () => <div data-test-subj="mock_CasesTable" />,
+}));
+
+jest.mock('../components/detection_response/host_alerts_table', () => ({
+  HostAlertsTable: () => <div data-test-subj="mock_HostAlertsTable" />,
+}));
+
 jest.mock('../components/detection_response/rule_alerts_table', () => ({
   RuleAlertsTable: () => <div data-test-subj="mock_RuleAlertsTable" />,
 }));
+
+jest.mock('../components/detection_response/user_alerts_table', () => ({
+  UserAlertsTable: () => <div data-test-subj="mock_UserAlertsTable" />,
+}));
+
 // TODO: add all sections mocks
```

Comment:
can you remove this `TODO` please

Reply:
Angela also left the TODOs 😂 . Okay, yes you can also remove them in the conflicts-solving when you merge both branches.

Reply:
lol no problem. I removed them from mine but will check again once all branches are in :P